tests/common/ResponseGenerator.cpp

```diff
@@ -19,6 +19,7 @@
 
 #include "ResponseGenerator.h"
 
+#include <algorithm>
 #include <string>
 #include <vector>
```

Comment:
It would be nice to clean excess inclusions too (e.g. '<vector>' is a quite massive one but clearly not needed here). Of course as a separate ticket.

Reply:
Agree. But to clear obsolete includes will be a bit more tricky. Make sense to add a ticket and do it lately